source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveCardRenderer.cs

```diff
@@ -288,15 +288,5 @@ private async Task<RenderedAdaptiveCardImage> RenderCardToImageInternalAsync(Ada
             await visitor.GetAllImages(card).WithCancellation(cancellationToken).ConfigureAwait(false);
             return visitor.LoadedImages;
         }
-
-        public void UseXceedElementRenderers()
-        {
-            throw new NotImplementedException();
-        }
-
-        public void UseDefaultElementRenderers()
-        {
-            throw new NotImplementedException();
-        }
```

Comment:
yes. The default element renderers are already configured.

It is only the Xceed renderer library which has 2 extension methods that allow you to Explicitly set the input renderers to use the default or the xceed renderers.

See AdaptiveCards.Rendering.Wpf.Xceed\XceedAdaptiveCardRenderer.cs

Defining the methods UseDefaultElementRenderers() UseXceedElementRenderers() makes no sense (because the WPF library has no dependency on Xceed) and worse defining those methods on the class disabled the extension methods, breaking the WPF visualizer because it is no longer calling the Xceeed extension methods.